src/pages/MenuPage.js

```diff
@@ -99,14 +104,12 @@ const Menu = ({
     () => (
       <View style={containerStyle}>
         <SupplierImage style={image} />
-        <View>
-          <InfoBadge routeName={ROUTES.SUPPLIER_INVOICES}>
-            <MenuButton text={navStrings.supplier_invoices} onPress={toSupplierInvoices} />
-          </InfoBadge>
-          <InfoBadge routeName={ROUTES.SUPPLIER_REQUISITIONS}>
-            <MenuButton text={navStrings.supplier_requisitions} onPress={toSupplierRequisitions} />
-          </InfoBadge>
-        </View>
+        <InfoBadge routeName={ROUTES.SUPPLIER_INVOICES}>
```

Comment:
"lol noobs whys this here"

![image](https://user-images.githubusercontent.com/35858975/104540101-b3ab8900-5683-11eb-9af1-83822469353e.png)

"ah .."

🤣 

I very much support cleaning up components like this - I am not sure of a different solution (Or more accurately, can't be bothered finding one? 🤣 ) other than putting em back!

Reply:
LOL

Reply:
I thought I should check modules but I was like nah, what's the worst that could happen 🤣

Reply:
Reverted commit https://github.com/openmsupply/mobile/pull/3333/commits/f58cfb768d405b31a3f65aacc6f73445b0f54647

Reply:
![image](https://user-images.githubusercontent.com/35858975/104542377-40584600-5688-11eb-9a19-1495bce8ad13.png)

Reply:
????

Reply:
Just putting true as the modules so that you get that layout/modules enabled without having to do it through msupply